std_msgs/Header header
float64 roll_ref
float64 pitch_ref
float64 yaw_rate_cmd
float64 thrust_ref